tests/unit/configFile.spec.js

```diff
@@ -133,4 +133,239 @@ describe("configFile.js", () => {
       expect(result).toStrictEqual(changedSettings);
     });
   });
+
+  describe("parseBasicSettings function", () => {
+    it("data should be parsed correctly", async () => {
+      await LocalStore.clear();
+      const expectedParsedData = {
+        configurationURL:
+          "https://raw.githubusercontent.com/AdvancedThreatAnalytics/threat-analytics-search/main/settings.json",
+        useGroups: true,
+        configEncrypted: false,
+        configEncryptionKey: "",
+        autoUpdateConfig: false,
+      };
+      const result = await ConfigFile.default.parseBasicSettings(
+        defaultSettings.config
+      );
+
+      expect(result).toEqual(expect.objectContaining(expectedParsedData));
+      expect(result.mergeGroups).toEqual(result.useGroups);
+    });
+  });
```

Comment:
Also these cases (described on the issue) weren't implemented:

* Boolean values expressed as string (e.g. "true" or "false") are converted to bool (e.g. true or false).
* The mergeGroups values is initialized with the same value than useGroups.